docs/get_started.rst

```diff
@@ -1,17 +1,388 @@
+
+If you're developing in persistent orgs and not already using source control, you'll also need to retrieve the existing metadata from the org. Note that adopting source control and moving from persistent orgs into scratch orgs are case-specific processes; your experience may vary and you may need to undertake additional work as part of those transitions.
+
+You're welcome to discuss project conversion in the `CumulusCI (CCI) Trailblazer group <https://trailblazers.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F9300000009M9Z>`_.
+
+
+
+Project Setup
+^^^^^^^^^^^^^
+
+Prior to retrieving your project's metadata you need to set up the project's root directory. These steps include:
+
+* Creating a project directory and initializing it as a Git repository as outlined in `starting a new CumulusCI project`_.
+* Initializing the directory as a CumulusCI project as outlined in `project initialization`_.
+
+
+
+Retrieving Metadata
+^^^^^^^^^^^^^^^^^^^
+
+The following assumes that your project currently lives in a persistent org, such as a Developer Edition org or a sandbox. We recommend retrieving metadata via the Metadata API (using the Salesforce CLI), followed by converting the source from Metadata API to SFDX format.
+
+#. If the metadata you want to retrieve isn't already in a package, `create one <https://help.salesforce.com/articleView?id=creating_packages.htm>`_.
+    .. note:: If your project contains a managed package, ensure that the package namespace matches the namespace you entered when running ``cci project init``.
+
+#. Run the `retrieve command <https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_force_mdapi.htm#cli_reference_retrieve>`_ to extract your package metadata::
+
+    $ sfdx force:source:retrieve -n package_name /path/to/project/
+
+That's it! You now have all of the metadata you care about in a single Git repository configured for use with CumulusCI.
+At this point you may want to `add your repo to GitHub`_, or perhaps begin `configuring CumulusCI` <#TODO doc ref>.
+
+
+
+Org Shape Setup
+^^^^^^^^^^^^^^^
+
+TODO: link to org section
+
+
+
+Other Conversion Considerations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Convert an existing package to CumulusCI
-----------------------------------------
+* If you or your team have been working with `scratch org definition files <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_def_file.htm>`_ for use with ``sfdx``, consult our documentation on `configuring orgs` <#TODO doc ref> to utilize them with CumulusCI.
+* If you have metadata that you would like deployed pre or post deployment, `#TODO <pre/post ref>`.
+* If you have data that you need to include for testing or production purposes, visit our section on `Automating Data Operations` <#TODO doc ref>.
```

Comment:
```suggestion
* If you have metadata that you would like deployed pre or post deployment, see `#TODO <pre/post ref>`.
* If you have data that you need to include for testing or production purposes, see `Automating Data Operations` <#TODO doc ref>.
```
For these straightforward pointers to other doc, IMO it's better to use the same text to lead into the linked docs. More readable, less cognitive demand.